docs/usage.md

```diff
@@ -154,6 +154,24 @@ includes:
 > This was a deliberate decision to keep use and implementation simple.
 > If you disagree, open an GitHub issue and explain your use case. =)
 
+### Optional includes
+
+Includes marked as optional will allow Task to continue execution as normal if
+the included file is missing.
```

Comment:
Perhaps actually write out in words the scenario below, so someone reading can understand the value?

Something like: "This is useful for removing task files providing development and testing commands from production environments".